Sources/PackageLoading/PackageDescription4Loader.swift

```diff
@@ -296,13 +296,23 @@ extension PackageDependencyDescription {
                 // If the URL has no scheme, we treat it as a path (either absolute or relative to the base URL).
                 return AbsolutePath(url, relativeTo: AbsolutePath(baseURL)).pathString
             }
+            
+            if case .localPackage = requirement {
+                do {
+                    return try AbsolutePath(validating: url).pathString
+                } catch PathValidationError.invalidAbsolutePath(let path) {
+                    throw ManifestParseError.invalidManifestFormat("'\(path)' is not a valid path. Please use a relative or absolute path for local dependencies.", diagnosticFile: nil)
```

Comment:
How about this:

`'\(path)' is not a valid path for path-based dependencies; use relative or absolute path instead`

Reply:
Sounds good 👍 Changed in b3bb906